vassal-app/src/main/java/VASSAL/build/module/GameState.java

```diff
@@ -540,11 +535,16 @@ else if (!saveModuleVersion.equals(moduleVersion)) {
         loadContinuation(f);
       }
       else {
-        //BR// New preferred style load for vlogs is close the old stuff and hard-reset to the new log state.
-        final boolean foreground = gameStarted;
         g.setGameFile(f.getName(), GameModule.GameFileMode.LOADED_GAME);
-        if (foreground) {
+
+        //BR// New preferred style load for vlogs is close the old stuff and hard-reset to the new log state.
+        if (gameStarted) {
+          GameModule.getGameModule().setGameFileMode(GameModule.GameFileMode.NEW_GAME);
+          GameModule.getGameModule().setLoadOverSemaphore(true); // Stop updating Map UI etc for a bit
+
```

Comment:
Can there be an exception between here and where you unset the semaphore? If so, it needs to be in a `finally`.